change/@ni-nimble-angular-6181d362-ba99-4697-9e75-7a8ed3d061ff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Just want to double check that you regenerated `package-lock.json` after resolving the changes from main. Github's rich diff shows a bit more churn than I would have predicted. If you did, no worries, I guess the new package just adds a lot of deps.

Reply:
Yep.
Specifically I undid the angular/localize change, got the package-lock from main, did npm install (verified package-lock still matched main at this point), redid the angular/localize change, and did npm install again:
`added 15 packages, changed 1 package, and audited 2566 packages in 5s`
I think the lines removing dev:true on some packages are probably dependencies of angular/localize.